dali/python/nvidia/dali/plugin/pytorch.py

```diff
@@ -284,3 +286,22 @@ def __init__(self,
                                                          size, auto_reset = auto_reset,
                                                          stop_at_epoch = stop_at_epoch,
                                                          dynamic_shape = dynamic_shape)
+
+
+class PyTorchFunction(ops.PythonFunction):
```

Comment:
```suggestion
class TorchPythonFunction(ops.PythonFunction):
```
It would be good to avoid suggesting that this is PyTorch function while it is python one in DALI that can operate on Torch tensors.

Reply:
Good point